runtime/lua/vim/filetype.lua

```diff
@@ -892,7 +892,6 @@ local filename = {
   [".gitconfig"] = "gitconfig",
   [".gitmodules"] = "gitconfig",
   ["/.config/git/config"] = "gitconfig",
-  ["/etc/gitconfig"] = "gitconfig",
```

Comment:
This looks correct but the line just above this looks suspicious. I think that should also be moved to a pattern style entry with `.*` at the beginning. Are you okay doing that as part of this PR or do you want to do it separately?